modules/cluster-logging-elasticsearch-limits.adoc

```diff
@@ -37,8 +37,28 @@ spec:
             cpu: "4000m"
             memory: "4Gi"
           requests:
-            cpu: "100m"
+            cpu: "4000m"
             memory: "1Gi"
 ----
 <1> Specify the CPU and memory limits as needed. If you leave these values blank,
 the Elasticsearch Operator sets default values that should be sufficient for most deployments.
++
+If you need to adjust the amount of Elasticsearch CPU and memory, you must change both the request value and the limit value. 
```

Comment:
> ! need to v
Avoid using “need to” to refer to a requirement that a product or procedure imposes. Use “must” or, in
constructions in which “must” is not idiomatically correct, use “have to” or “required to.”

Maybe ~If you need~